libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -37,11 +36,11 @@ internal class RouteRefreshController(
         private const val FAILED_ATTEMPTS_TO_INVALIDATE_EXPIRING_DATA = 3
     }
 
-    suspend fun refresh(routes: List<NavigationRoute>): List<NavigationRoute> {
+    suspend fun refresh(routes: List<NavigationRoute>, legIndex: Int): List<NavigationRoute> {
```

Comment:
why is it a parameter now?

Reply:
To avoid the possibility of indices (route_geometry and current_leg) being out of sync. Is that a problem? We would just refresh all routes starting with the same leg.The only bad thing that can happen is the request being slightly less optimal, but it wouldn't be noticeable if we provide current_route_geometry_index. Right?

Reply:
I removed it.